extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1088,6 +1090,9 @@ export class AmpA4A extends AMP.BaseElement {
           // modified url.
           'src': xhrFor(this.win).getCorsUrl(this.win, adUrl),
         }, SHARED_IFRAME_PROPERTIES));
+    const sentinel = generateSentinel(window);
+    const metadata = getContextMetadata(window, iframe, sentinel);
+    iframe.setAttribute('name', encodeURIComponent(JSON.stringify(metadata)));
```

Comment:
because this is getting written to the name attribute in the html, this was to protect against if anything in the metadata has a closing quote of any kind